clang/test/Modules/specializations-lazy-load-parentmap-crash-analyzer.cpp

```diff
@@ -0,0 +1,87 @@
+// REQUIRES: staticanalyzer
+//
```

Comment:
```suggestion
```
My understanding is that we can now enable this part of the test also when `CLANG_ENABLE_STATIC_ANALYZER=OFF`, right?